src/api/index.ts

```diff
@@ -7,20 +7,28 @@ export interface Fraction {
 }
 
 export interface CourseItem {
+	id: string;
 	name: string;
 	weight: number;
-	grade?: Grade;
+	grade?: number | Fraction;
 	dueDate?: Date;
 }
 
 export interface Course {
+	id: string;
 	name: string;
 	crn?: number;
```

Comment:
I don't think we have a `crn` field in the backend 😮 should we?

Reply:
There was mention of offering the ability to enter a course CRN and share the syllabus amongst other users, but at this time we aren't implementing this feature. Maybe not now but later we can add it.